src/main/java/com/xiaomi/infra/pegasus/rpc/async/ReplicaSessionHookManager.java

```diff
@@ -0,0 +1,23 @@
+package com.xiaomi.infra.pegasus.rpc.async;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReplicaSessionHookManager {
```

Comment:
I would suggest naming this class a `SessionInterceptor` to keep aligned with `TableIntercetor`. And you can place this class in `interceptor/` package.

Reply:
Yes, I can put it in `interceptor/`, but there would be two interceptorManager, which are `InterceptorManager` and `ReplicaSessionInterceptorManager`